numpy/core/src/umath/ufunc_object.h

```diff
@@ -12,6 +12,9 @@ ufunc_seterr(PyObject *NPY_UNUSED(dummy), PyObject *args);
 NPY_NO_EXPORT const char*
 ufunc_get_name_cstr(PyUFuncObject *ufunc);
 
+NPY_NO_EXPORT PyObject *
+PyUFunc_GetDefaultIdentity(PyUFuncObject *ufunc, npy_bool *reorderable);
+
 /* strings from umathmodule.c that are interned on umath import */
```

Comment:
Is this "get reduction default" for iterating over empty arrays or "get reduction initial"?

Reply:
Neither quite unfortunately :(.  Because it is the initial always *except* for object arrays.